### ISSUE TITLE
Trend figure in readme

### ISSUE THREAD

**Issue by strengejacke** [easystats/easystats#430]
@easystats/core-team

The trend figure in the readme (https://easystats.github.io/easystats/#trend) shows daily downloads. Since the easystats projects now exists for more than 5 years, the figure became very unclear and chaotic, see:

![image](https://github.com/user-attachments/assets/3c478213-9cb1-4dd7-803a-7b52f169030d)

I now changed the code that takes the daily averages per month, so the lines are a bit smoother:

![image](https://github.com/user-attachments/assets/b3f347ea-2b72-4f5b-bf51-5109ef3c7288)

Still, we may consider changing to total downloads per months, not per day? We now have a time span from over 120 months, do we still need the "daily" units? This plot looks similar to what we have now, it's just a different unit:

![image](https://github.com/user-attachments/assets/ca1f7be7-9798-4661-be97-04cbb95570b4)

**Comment by strengejacke** [comment]
oh man, crazy that our packages almost have 23 millions downloads on CRAN 🚀

**Comment by IndrajeetPatil** [comment]
I would vote for the monthly downloads option.

**Comment by IndrajeetPatil** [comment]
> oh man, crazy that our packages almost have 23 millions downloads on CRAN 🚀 

Thanks you, _sjPlot_, _sjmisc_, _ggeffects_, etc. 😆

**Comment by strengejacke** [comment]
haha, but no, except _ggeffects_, there weren't many updates in the past months to my other packages. And indeed, _insight_, but also _parameters_ and _performance_ are indeed imported(!) by foreign packages outside our projects.

**Comment by rempsyc** [comment]
Yes, let's go for monthly downloads.

Also, I'm trying to understand our download statistics in context. `ggplot2` for example has been downloaded 145 million times. But the only estimates I know of evaluate that there are only 1-2 million R users worldwide. Assuming that number is constant, it would mean each user updated `ggplot2` 70 or so times?

How many R users are there now in 2024, since it's probably exponentially increased since the last estimates? What's actually driving all these CRAN downloads? How much is automatic and how much reflects true usage?

**Comment by DominiqueMakowski** [comment]
To be honest I kinda the daily patterns because it shows the yeaely Christmas dips (and stuff like COVID etc.) and I find it fascinating 

But yeah, monthly is fine :)

Why is datawizard and insight rates going down though 🤔

**Comment by etiennebacher** [comment]
> oh man, crazy that our packages almost have 23 millions downloads on CRAN 🚀

I would really like to know the share that comes from CI: I think for every commit in every repo, we install `insight` 15-20x times. There's also the fact that we have more commits that Github displays since we squash all commits before merging them to `main`.

---

Fine with whatever dataviz you pick ;)

**Comment by IndrajeetPatil** [comment]
Only [PPM](https://packagemanager.posit.co/client/#/) can help us estimate this. 

The rule of thumb would be that the majority of downloads on personal computers will be tracked by macOS and Windows, while the automated downloads will be tracked by Linux distros (since most servers use this OS).

**Comment by strengejacke** [comment]
> To be honest I kinda the daily patterns because it shows the yeaely Christmas dips (and stuff like COVID etc.) and I find it fascinating

Yeah, such extreme peaks are now a bit more "flattened". But the code chunk is just not evaluated, it's not removed from the readme, so you can get back to it whenever you (we) like :-)

> Why is datawizard and insight rates going down though 🤔

![image](https://github.com/user-attachments/assets/82c2d0a1-c2d7-4ce6-9e35-1139668e8b89)

It's not going much down (summer time), it's just that for this month we don't have the latest download stats. That's probably the decrease you find in every figure (less so for daily downloads, I guess).

**Comment by bwiernik** [comment]
Agree with monthly

**Comment by mattansb** [comment]
I suggest also sorting the legend by total downloads + removing the linear trend lines

**Comment by ChirraneZakariya** [comment]
Hello. I am new to the package and together with @aboudiallo21 and
under the supervision of [@katossky / @JMeyer31] we are willing to solve this
issue, as part of an assignment at the ENSAI school of statistical engineering
(France). We have read trend figure in readme and we will try to be as
autonomous as possible but we may ask for guidance from now and then. We
have a solid background in statistics and a reasonable one in computer science.
If any maintainer thinks this issue is too ambitious for a beginner, please tell
us so and - if you have time - guide us to a project / issue were our help may
be more useful.
